lib/govuk_index/publishing_event_processor.rb

```diff
@@ -0,0 +1,41 @@
+require 'govuk_index/elasticsearch_presenter'
+
+module GovukIndex
+  class PublishingEventProcessor
+    TIMEOUT_SECONDS = 5.0
+
+    def initialize(index_name)
+      @index_name = index_name
+    end
+
+    def process(message)
+      document = ElasticsearchPresenter.new(message.payload).call
+      save_to_elastic_search(document)
+      message.ack
+    rescue Indexer::ValidationError => e
```

Comment:
I thought the plan was to avoid reuse of the `Indexer` namespaced code? Any reason we have declared the error class there?